src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const total = (cost * days);
+  const largeDiscount = 50;
+  const smallDiscount = 20;
+  const largeDiscountReq = 7;
+  const smallDiscountReq = 3;
+
+  if (days < largeDiscountReq && days >= smallDiscountReq) {
```

Comment:
Maybe we can avoid using `&&` in if, after reorganising them. What if we checked first for condition when there is no discount, then small discount and then the large one?